packages/platform-core/src/usage-data/constants.ts

```diff
@@ -7,3 +7,7 @@ export const latestApiVersion = 'v1.0';
  * returns the latest available payload version
  */
 export const latestPayloadVersion = '1.1.0';
+
+export const TELEMETRY_ENABLED_KEY = 'telemetry.enabled';
+export const TELEMETRY_ANONYMOUS_ID = 'telemetry.anonymousId';
+export const TELEMETRY_CONFIG_KEY = 'telemetry';
```

Comment:
Are we using this?

Reply:
Yea we are using these constants to retrieve values from config in usage_data_emitter_factory & configure_telemetry.command

Reply:
I meant this key `TELEMETRY_CONFIG_KEY` I couldn't find where it was being read